common/sgx/tcbinfo.c

```diff
@@ -240,6 +240,16 @@ static oe_tcb_level_status_t _parse_tcb_status(
         status.fields.qe_identity_out_of_date = 1;
         status.fields.configuration_needed = 1;
     }
+    else if (_json_str_equal(str, length, "SWHardeningNeeded"))
+    {
+        status.fields.up_to_date = 1;
```

Comment:
>status.fields.up_to_date = 1; [](start = 8, length = 29)

Please add comment here on why SWHardeningNeeded is considered up_to_date.

Reply:
Done